openedx/features/content_tagging/rest_api/v1/tests/test_views.py

```diff
@@ -0,0 +1,368 @@
+"""
+Tests tagging rest api views
+"""
+
+from unittest import mock
+import ddt
+from django.contrib.auth import get_user_model
+from django.urls import reverse
+from rest_framework import status
+from rest_framework.test import APITestCase
+
+from common.djangoapps.student.auth import add_users, update_org_role
+from common.djangoapps.student.roles import CourseCreatorRole, OrgContentCreatorRole
+from organizations.models import Organization
+from openedx.features.content_tagging.models import Taxonomy
+
+User = get_user_model()
+
+
+# ToDo: import from openedx
```

Comment:
I create the mixins in the openedx-learning, but I need to move the test folder structure to live inside `openedx_tagging` to import here, right?

Reply:
Ahh.. ya, no, we shouldn't do that, sorry. I forgot that openedx-learning has its `tests` dir outside of the distribution. But feel free to borrow/copy/paste code from there as you wish.